client/src/page/compositePage.jsx

```diff
@@ -17,27 +18,28 @@ import SessionEnumerationConfigPage from './tool/sessionEnumerationConfig';
  * We can revisit this pattern later.
  */
 class CompositePage extends React.Component {
-
-    render() {
-        let url = new URL(window.location.href);
-        if (url.pathname === '/StaffVolunteerPage.php') {
-            return (<StaffVolunteerPage />);
-        } else if (url.pathname === '/AdminModules.php') {
-            return (<AdminModulesPage />);
-        } else if (url.pathname === '/volunteering.php') {
-            return (<VolunteerSignUpPage />);
-        } else if (url.pathname === '/brainstorm.php') {
-            return (<BrainstormPage />);
-        } else if (url.pathname === '/assignParticipants.php') {
-            return (<AssignmentsPage />);
-        } else if (url.pathname === '/assignSessionNumberConfig.php') {
-            return (<SessionEnumerationConfigPage />);
-        } else if (url.pathname === '/printRoomScheduleConfig.php') {
-            return (<PrintRoomScheduleConfigPage />);
-        } else {
-            return (<h4>Unknown Page</h4>);
-        }
+  render() {
+    let url = new URL(window.location.href);
+    if (url.pathname === "/StaffVolunteerPage.php") {
+      return <StaffVolunteerPage />;
+    } else if (url.pathname === "/AdminModules.php") {
+      return <AdminModulesPage />;
+    } else if (url.pathname === "/volunteering.php") {
+      return <VolunteerSignUpPage />;
+    } else if (url.pathname === "/brainstorm.php") {
+      return <BrainstormPage />;
+    } else if (url.pathname === "/assignParticipants.php") {
+      return <AssignmentsPage />;
+    } else if (url.pathname === "/assignSessionNumberConfig.php") {
+      return <SessionEnumerationConfigPage />;
+    } else if (url.pathname === "/printRoomScheduleConfig.php") {
+      return <PrintRoomScheduleConfigPage />;
+    } else if (url.pathname === "/devLogin.php") {
+      return <DevLoginPage />;
+    } else {
+      return <h4>Unknown Page</h4>;
     }
+  }
 }
```

Comment:
It looks like your VS Code is automatically changing the indent to 2 spaces, whereas other code is set to four spaces. That makes the PR a bit noisier than it needs to be.